ui/base-tool/src/main/res/drawable/ic_swap.xml

```diff
@@ -0,0 +1,20 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this resource should have been moved from the `tract-renderer` module to the `base-tool` module